docs/en/stack/ml/get-started-trial.asciidoc

```diff
@@ -8,13 +8,13 @@ image::images/management-license.png["The License Management page in {kib}"]
 For more information about Elastic license levels, see 
 https://www.elastic.co/subscriptions.
 
-You can start a 30-day trial to try out all of the platinum features, including 
-{ml-features}. Click **Start trial** on the **License Management** page in {kib}.
+You can start a 30-day trial to try out all of the subscription features,
```

Comment:
We could reiterate `30-day trial for free` (or `free trial`), since this is an important point to mention.